File: scariestMask.js
function scariestMask(mask) {
    let count = 0;
    for (let i = 0; i < mask.length; i++) {

        let min = 0;
        let max = mask[i].length - 1;

        while (min < max) {
            count += mask[i][min] != mask[i][max] ? 1 : 0

            min++
            max--;
        }
    }
    return count;
}

File: scariestMask.py
def scariestMask(mask):
    count = 0
    for i in range(len(mask)):
        low = 0
        high = len(mask[i]) - 1

        while low < high:
            if mask[i][low] != mask[i][high]:
                count += 1
            low += 1
            high -= 1
            pass
    return count
